conf/application.conf

```diff
@@ -126,6 +126,7 @@ microservice {
     auth {
       host = localhost
       port = 8500
+      confidenceLevel = 250
```

Comment:
there is already confidenceLevel in the config

Reply:
Done

Reply:
this must be 200, as we are not lifting CL to 250 in this iteration